txmongo/gridfs.py

```diff
@@ -3,7 +3,6 @@
 # found in the LICENSE file.
 
 from __future__ import absolute_import, division
-
 from txmongo._gridfs import GridFS, GridIn, GridOut, errors, grid_file, GridOutIterator
 
 assert GridFS
```

Comment:
Heh... the asserts are for silly coverage reasons but aren't necessary. NOOPs.

I honestly want to either get rid of gridfs or if it is used by anyone (hello?!) refactored.